schema.sql

```diff
@@ -0,0 +1,175 @@
+-- MySQL Script generated by MySQL Workbench
+-- Sat Oct 31 16:57:47 2020
+-- Model: New Model    Version: 1.0
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION';
+
+-- -----------------------------------------------------
+-- Schema yeticave_db
+-- -----------------------------------------------------
+
+-- -----------------------------------------------------
+-- Schema yeticave_db
+-- -----------------------------------------------------
+CREATE SCHEMA IF NOT EXISTS `yeticave_db` DEFAULT CHARACTER SET utf8 ;
+USE `yeticave_db` ;
+
+-- -----------------------------------------------------
+-- Table `yeticave_db`.`category`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `yeticave_db`.`category` (
+                                                        `category_id` INT NOT NULL AUTO_INCREMENT,
+                                                        `category_title` VARCHAR(100) NOT NULL,
+                                                        `symbolic_code` VARCHAR(100) NOT NULL,
+                                                        PRIMARY KEY (`category_id`),
+                                                        UNIQUE INDEX `title_UNIQUE` (`category_title` ASC) VISIBLE)
+    ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `yeticave_db`.`item`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `yeticave_db`.`item` (
+                                                    `item_id` INT NOT NULL AUTO_INCREMENT,
+                                                    `creation_date` TIMESTAMP(14) NOT NULL,
```

Comment:
[ERROR in query 7] Too-big precision 14 specified for 'creation_date'. Maximum is 6.